README.md

```diff
@@ -557,6 +557,16 @@ data class BlackjackHand(
 )
 ```
 
+or for the enum classes:
```

Comment:
Can you add more here? This is vague on its own and should actually go on the proguard rules section

Reply:
I'm afraid I don't have enough knowledge to be sure of what it changes compared to a simple @Keep, I had the feeling it was more than just proguard.

Reply:
I mean just make this a full sentence and put it in the correct proguard section of the readme